public/partials/navigation-admin.html

```diff
@@ -0,0 +1,61 @@
+<div id="sidebar-menu" style="width: 263px;">
+  <ul id="nav" class="nav collapse in">
+    <li class="panel">
+      <a href="#" class="selected">Home</a>
+    </li>
+    <li class="panel">
+      <a href="#products-sub" data-toggle="collapse" data-parent="#nav">Users<span class="pull-right glyphicon glyphicon-menu-down" aria-hidden="true"></span></a>
+      <ul id="products-sub" class="collapse in nav">
+        <li class="panel">
+          <a href="#billeder-sub" data-toggle="collapse" data-parent="#products-sub">User Acounts <span class="caret"></span></a>
+          <ul id="billeder-sub" class="collapse nav">
+            <li><a href="#">Sub 1</a></li>
+            <li><a href="#">Sub 2</a></li>
+            <li><a href="#">Sub 3</a></li>
+          </ul>
+        </li>
+        <li class="panel">
+          <a href="#bolig-sub" data-toggle="collapse" data-parent="#products-sub">User Activities <span class="caret"></span></a>
+          <ul id="bolig-sub" class="collapse nav">
+            <li><a href="#">Sub 1</a></li>
+            <li><a href="#">Sub 2</a></li>
+            <li><a href="#">Sub 3</a></li>
+          </ul>
+        </li>
+        <li>
+          <a href="#">Just another Option</a>
+        </li>
+      </ul>
+    </li>
+
+    <li class="panel">
+      <a href="#brands-sub" data-toggle="collapse" data-parent="#nav">Apps<span class="pull-right glyphicon glyphicon-menu-down" aria-hidden="true"></span></a>
+      <ul id="brands-sub" class="collapse nav">
+        <li>
+          <a href="#">Developer Apps</a>
+        </li>
+        <li>
+          <a href="#">App Activities</a>
+        </li>
+        <li>
+          <a href="#">App Logs</a>
+        </li>
+      </ul>
+    </li>
+    <li class="panel">
+      <a href="#" data-toggle="collapse" data-parent="#nav">Roles<span class="pull-right" aria-hidden="true"></span></a>
+    </li>
+  </ul>
+</div>
+
+<script>
```

Comment:
@thinkking remove the script into a separate file.  Reference that file within the partials if necessary, all control logic should be separated from views to allow for easy maintenance of the platform